docs/docsite/rst/user_guide/playbooks_filters.rst

```diff
@@ -47,6 +47,22 @@ for example::
     - set_fact:
         myvar: "{{ result.stdout | from_json }}"
 
+
+.. versionadded:: 2.7
+
+To parse multi-document yaml strings, the 'from_yaml_all' filter is provided.
```

Comment:
Wrap 'from_yaml_all' in double backticks rather than single quotes in order for it to be formatted as inline code.